File: beecrowd/beginner/c/1134.c
#include <stdio.h>

int main() {
    int op, alcohol = 0, gasoline = 0, diesel = 0;

    while (1) {
        scanf("%d", &op);

        if (op == 4) break;

        if (op == 1) {
            alcohol++;
        } else if (op == 2) {
            gasoline++;
        } else if (op == 3){
            diesel++;
        } else {
            continue;
        }
        
    }

    printf("MUITO OBRIGADO\n");
    printf("Alcool: %d\n", alcohol);
    printf("Gasolina: %d\n", gasoline);
    printf("Diesel: %d\n", diesel);

    return 0;
}

File: beecrowd/beginner/python/1134.py
alcohol = gasoline = diesel = 0

while 1:
    op = int(input())

    if op == 4:
        break

    if op < 1 or op > 3:
        continue

    match op:
        case 1:
            alcohol += 1
        case 2:
            gasoline += 1
        case 3:
            diesel += 1

print("MUITO OBRIGADO")
print(f"Alcool: {alcohol}")
print(f"Gasolina: {gasoline}")
print(f"Diesel: {diesel}")
